src/Examples/TensorboardExample/Tensorboard.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+using System.IO;
+using System.Linq;
+using SkiaSharp;
+using static TorchSharp.torch;
+
+namespace TorchSharp.Examples.TensorboardExample
+{
+    public static class Tensorboard
+    {
+        internal static void Main(string[] args)
```

Comment:
Great examples!! Love it. Let's try to download images at the start, like we do with data sets. That should avoid any copyright issues, I think.